### DISCUSSION TITLE
Support JSON file upload for bulk insert of records

### DISCUSSION THREAD

**Discussion by kdhrubo** [9tigerio/db2rest#612] [Ideas]
1. Max file size of 5MB to be supported. 
2. This will be an async operation.
3. Records will be streamed in chunks.

**Comment by MichalisDBA** [comment]
Hi @kdhrubo.

Why 1. Max File size will be 5MB only?
Will it be configurable to increase the limit?

**Comment by kdhrubo** [comment]
The larger the file size it will take longer to complete the operation.
There will be a configuration to adjust the size limit.
